common/converters_test.go

```diff
@@ -329,3 +329,35 @@ func TestShardAssignment(t *testing.T) {
 		fmt.Printf("Shard %d:\n\t\t%d accounts\n", sh, cnt)
 	}
 }
+
+func TestProcessDestinationShardAsObserver(t *testing.T) {
+	t.Parallel()
+
+	shard, err := common.ProcessDestinationShardAsObserver("")
```

Comment:
split in more t.Run tests to specify what we test?

Reply:
updated